### ISSUE TITLE
Setting up remote standalone instance fails with endless waiting

### ISSUE THREAD

**Issue by Quix0r** [payara/Payara#1716]
# Description #

I have followed [this tutorial](http://blog.payara.fish/payara-server-basics-creating-a-simple-cluster) and I'm now at the said step (**Creating a Remote Standalone Instance**).

I have entered/choosen following:
Instance Name: **zulu289-instance**
Node: **zulu289**
Configuration: **cluster-config**

The said node zulu289 is setup as SSH tunnel and "ping" is successful, no "force" was required as I used a password-less SSH key to connect to the instance which went smoothly.

## Expected Outcome

The step should finish within a moments as I have a good Internet connection (1 MByte/sec upload).

## Current Outcome

The request remains unfinished and stays forever there. No instance is being created.

## Environment ##

- **Payara Version**: 4.1.1.172
- **Edition**:Full
- **JDK Version**: OpenJDK 7/8
- **Operating System**: Linux
- **Database**: _unrelevant_

Local instance:
openjdk version "1.8.0_91"
OpenJDK Runtime Environment (build 1.8.0_91-8u91-b14-3-b14)
OpenJDK 64-Bit Server VM (build 25.91-b14, mixed mode)

zulu289 instance:
java version "1.7.0_111"
OpenJDK Runtime Environment (IcedTea 2.6.7) (7u111-2.6.7-2~deb7u1)
OpenJDK 64-Bit Server VM (build 24.111-b01, mixed mode)

**Comment by mikecroft** [comment]
Does the target directory structure exist, and does the user you are using have the correct permissions to write to the directory or to create the structure?

You could try to SCP a file across to the remote machine using the same credentials and targetting the same directory to prove that. Another thing to try would be to start a local virtual machine and recreate the problem to rule out the possibility of any network issue.

What would be really helpful here is [a thread dump (or a couple of thread dumps)](http://blog.payara.fish/taking-a-thread-dump-of-payara-server-part-1) while the request is hanging. We should then be able to spot which thread is hanging and get a stack of what it was doing while it was hanging.

Can you try the above please?

**Comment by mikecroft** [comment]
Hi @Quix0r have you been able to try the troubleshooting steps above? It's not clear whether this is a bug or misconfiguration at the moment and we can't progress the issue without more data.

**Comment by Quix0r** [comment]
Not yet, to busy weeks. :-(

**Comment by Quix0r** [comment]
SSH/SCP works flawless, so that cannot be the case.

**Comment by Quix0r** [comment]
Thread dumps uploaded:
https://cloud.mxchange.org/index.php/s/149vSQYmDeENiuY/download?path=%2FPayara&files=thread-dump001.txt
https://cloud.mxchange.org/index.php/s/149vSQYmDeENiuY/download?path=%2FPayara&files=thread-dump002.txt
https://cloud.mxchange.org/index.php/s/149vSQYmDeENiuY/download?path=%2FPayara&files=thread-dump003.txt

**Comment by Quix0r** [comment]
Also the target directory exists. I will retry it with latest stable build, if time permits. ;-)

**Comment by smillidge** [comment]
Is this still an issue?

**Comment by Quix0r** [comment]
Need to retest then. Thank you for the head up.

**Comment by Quix0r** [comment]
I have followed this post: http://blog.payara.fish/creating-a-simple-cluster-with-payara-server for the second instance I have choosen my server's node (not local again). Now it hangs very long. So still an issue, I guess?

**Comment by Quix0r** [comment]
Maybe I made something wrong?

1) I have created under "Nodes" another node (my remote server) with SSH and proper login details (password-less SSH key) and `payara` for the username as this is the username on remote site.
2) I created a cluster with name "home" and choose to duplicate the default-config.
3) I created an instance, referencing my local node and referencing home-config.
4) Then I created another instance referencing my server node and also referencing home-config
5) Then it hangs.
